programs/nft-stickers/src/lib.rs

```diff
@@ -0,0 +1,350 @@
+use anchor_lang::prelude::*;
+use solana_program::{
+    program::{invoke, invoke_signed},
+    program_error::ProgramError,
+    msg};
+use anchor_spl::token::Token;
+use mpl_token_metadata::*;
+use spl_token;
+
+declare_id!("3T7Zj9GsgV6Mj1y3kYBKTkzJ3pQCY2FooFztCtVfpK2X");
+
+//const NFT_STICKER_PACK_PDA_SEED: &[u8] = b"nft-sticker-pack";
+const RELAYER_SEED: &[u8] = b"relayer";
+//pub const PREFIX: &[u8] = b"metadata";
+//pub const EDITION: &[u8] = b"edition";
+//pub const EDITION_MARKER_BIT_SIZE: u64 = 248;
+//const DISCRIMINATOR_LENGTH: usize = 8;
+//const STRING_LENGTH_PREFIX: usize = 4;
+//const STRING_LENGTH_URI: usize = 128;
+
+
+#[derive(Clone, AnchorSerialize, AnchorDeserialize, PartialEq)]
+pub struct Sticker {
+    /// Uri of a single sticker
+    pub uri: String,
+}
+
+#[program]
+pub mod nft_stickers {
+    use super::*;
+
+    /*pub fn create_sticker_pack(_ctx: Context<CreateStickerPack>) -> Result<()> {
+        msg!("Tutto ok per ora");
+        Ok(())
+    }*/
+
+    pub fn stake_master_edition(ctx: Context<StakeMasterEdition>) -> Result<()> {
+        
+        let metadata_info = ctx.accounts.metadata.as_ref();
+        let metadata = mpl_token_metadata::state::Metadata::from_account_info(metadata_info)?;
+        let update_authority = ctx.accounts.update_authority.as_ref();
+        let mint_info = ctx.accounts.mint.as_ref();
+        let master_edition_info = ctx.accounts.master_edition.as_ref();
+        let user_associated_token_account = ctx.accounts.associated_token_account.as_ref();
+        let pda_relayer_account = &ctx.accounts.relayer;
+        let token_program = ctx.accounts.token_program.as_ref();
+
+        if !update_authority.is_signer || (metadata.update_authority != *update_authority.key) {
+            return Err(ProgramError::MissingRequiredSignature.into());
+        }
+        if metadata.mint != *mint_info.key {
+            return Err(error!(ErrorCode::MetadataMismatch));
+        }
+
+        let master_edition = &mpl_token_metadata::state::MasterEditionV2::from_account_info(master_edition_info)?;
+        if master_edition.key != mpl_token_metadata::state::Key::MasterEditionV2 {
+            return Err(error!(ErrorCode::WrongEdition));
+        }
+        let mut found = false;
+        if let Some(creators) = metadata.data.creators {
+            for i in 0..creators.len() {
```

Comment:
Not sure the type but assuming its a slice or a vector you could do something like 

```rust
for creator in creators {
   if creator.address == pda_relayer_account.key() {
        found = true; 
        // could break here since its found?
   }
}
```